core/storage/base_model/gae_models.py

```diff
@@ -115,32 +117,34 @@ def get_multi(cls, entity_ids, include_deleted=False):
 
     @classmethod
     def put_multi(cls, entities):
-        """Stores sequence of entities.
+        """Stores the given ndb.Model instances.
 
         Args:
-            entities: sequence.
+            entities: list(ndb.Model).
 
-        Return:
-            list with the stored keys.
+        Returns:
```

Comment:
Ah, interesting. I don't actually think we want to expose ndb.Key to the rest of the application, since we generally deal with ids throughout. Can we change this method to not return anything without affecting behaviour in the rest of the app?